src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -569,6 +569,16 @@ type VehicleRentalStation implements Node & PlaceInterface {
     """
     spacesAvailable: Int
 
+    """
+    Number of vehicles currently available on the rental station, grouped by vehicle type.
+    """
+    vehicleTypesAvailable: [RentalVehicleTypeCount!]
+
+    """
+    Number of free spaces currently available on the rental station, grouped by vehicle type.
+    """
+    vehicleSpacesAvailable: [RentalVehicleTypeCount!]
+
```

Comment:
I think this is a good way of expressing the information but we will discuss this in one of our meeting if we can do better. (We leave ourselves lots of time to think about API schemas because once they are released it's almost impossible to change them later.)

Reply:
I think I would model this something like this (names are up for discussion):
<img width="634" alt="image" src="https://github.com/opentripplanner/OpenTripPlanner/assets/5525340/bf71bccd-629f-4690-8ed2-01d63fc0dc2b">